castai/resource_workload_scaling_policy.go

```diff
@@ -567,13 +680,37 @@ func toWorkloadScalingPoliciesMap(p sdk.WorkloadoptimizationV1ResourcePolicies)
 		m[FieldLimitStrategy] = []map[string]any{limit}
 	}
 
+	if v, ok := previousCfg["apply_threshold"].(float64); ok && v > 0 {
+		m["apply_threshold"] = p.ApplyThreshold
+	} else {
+		strategy := applyThresholdStrategyToMap(*p.ApplyThresholdStrategy)
```

Comment:
should we check for nil here 🤔 if I get it correctly, BE always return it, but in case of bug, maybe it's better to not panic here